### DISCUSSION TITLE
How to become tracker?

### DISCUSSION THREAD

**Discussion by d47081** [zeronet-conservancy/zeronet-conservancy#273] [Q&A] [answered]
Just launched ZNC on VPS, have Clearnet IP + Yggdrasil instance there.

Thoughts participate network distributions, it's dynamical DHT or hardcoded?

For right now opened just `39955` port for files exchange

**Comment by caryoscelus** [comment, marked as answer]
afaik easiest way is to enable Bootstrapper plugin

> it's dynamical DHT

i've attempted to add DHT [support](https://github.com/zeronet-conservancy/zeronet-conservancy/pull/255), it seemed to work on the surface, but need proper testing in real life situation

**Comment by slrslr** [comment]
It has been already answered, but if anyone needs more details, then search for "tracker" in [this zeronet tutorial](http://127.0.0.1:43110/1LfvE91ZF18jdG3wW62Dw7NtfTZh737KPL/?Topic:1747042241_1L4dZcDF2maSKHDy788yhxpYnBWnXadUtS). It should work not only for Tor .onion, but also other kind of protocols, including possibly [Yggdrasil one](http://127.0.0.1:43110/1LfvE91ZF18jdG3wW62Dw7NtfTZh737KPL/?Topic:1636624775_1L4dZcDF2maSKHDy788yhxpYnBWnXadUtS/Yggdrasil+encrypted+P2P+network).

**Reply by oooo-ps** [reply]
it would be useful to publish these links somewhere in the Internet..

**Reply by slrslr** [reply]
> publish these links somewhere in the Internet..

done
